System.Device.Dac/System.Device.Dac.nfproj

```diff
@@ -64,11 +64,12 @@
       <ProjectConfigurationsDeclaredAsItems />
     </ProjectCapabilities>
   </ProjectExtensions>
-  <Import Project="..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.1.91\build\Nerdbank.GitVersioning.targets')" />
+  <Import Project="..\packages\Nerdbank.GitVersioning.3.5.113\build\Nerdbank.GitVersioning.targets" Condition="Exists('..\packages\Nerdbank.GitVersioning.3.5.113\build\Nerdbank.GitVersioning.targets')" />
```

Comment:
Sorry Robin: this one is missing the props...

Reply:
Are you sure? I have just checked against `Spi` and `Collections` and the O/P at this line is the same...

Reply:
Ok... I have found what you mean.
will add in a min,

Reply:
Pretty sure! I've just fixed it.